src/sqlparser/src/parser.rs

```diff
@@ -2360,17 +2360,32 @@ impl Parser {
     }
 
     pub fn parse_explain(&mut self, describe_alias: bool) -> Result<Statement, ParserError> {
-        let analyze = self.parse_keyword(Keyword::ANALYZE);
-        let verbose = self.parse_keyword(Keyword::VERBOSE);
-        let trace = self.parse_keyword(Keyword::TRACE);
-
+        let mut analyze = false;
+        let mut distsql = false;
+        let mut verbose = false;
+        let mut trace = false;
+        while let Some(keyword) = self.parse_one_of_keywords(&[
+            Keyword::ANALYZE,
+            Keyword::DISTSQL,
+            Keyword::VERBOSE,
+            Keyword::TRACE,
+        ]) {
+            match keyword {
+                Keyword::ANALYZE => analyze = true,
+                Keyword::DISTSQL => distsql = true,
+                Keyword::VERBOSE => verbose = true,
+                Keyword::TRACE => trace = true,
+                _ => unreachable!("{}", keyword),
```

Comment:
Please handle this branch.

Reply:
this branch should not be executed after parse_one_of_keywords